docs/core/runtime-discovery/troubleshoot-app-launch.md

```diff
@@ -100,6 +100,62 @@ From the **Installers** column of the runtime download, download the installer m
 Different Linux distributions provide .NET through different package managers. See [Install .NET on Linux](../install/linux.md) for details. Note that preview versions of .NET are not available through package managers.
 ::: zone-end
 
+#### Run the dotnet-install script
+
+Download the [dotnet-install script](../tools/dotnet-install-script.md#recommended-version) for your operating system. Run the script with options based on the information in the error message. The [dotnet-install script reference page](../tools/dotnet-install-script.md) shows all available options.
+
+::: zone pivot="os-windows"
+Launch [PowerShell](/powershell) and run:
+
+```powershell
+dotnet-install.ps1 -Architecture <architecture> -InstallDir <directory> -Runtime <runtime> -Version <version>
+```
+
+For example, the error message in the section above would correspond to:
+
+```powershell
+dotnet-install.ps1 -Architecture x64 -InstallDir "C:\Program Files\dotnet\" -Runtime dotnet -Version 5.0.15
```

Comment:
I ran into a problem with a similar command yesterday. Apparently passing in `-Runtime aspnet` will install both `Microsoft.NETCore.App` and `Microsoft.AspNetCore.App` - which is good. But passing in `-Runtime windowsdesktop` will ONLY install `Microsoft.WindowsDesktop.App` - so one doesn't even get `dotnet.exe` for example (or `hostfxr`).

I filed https://github.com/dotnet/install-scripts/issues/266 against the scripts... not sure if we should mention it here though.

Reply:
Sad. I went back and forth, but I think I'm going to go with not mentioning in here unless we see confusion around it. The user should already have some form of the runtime installed for us to indicate a framework is not found, so I think worst case is the user does `-Runtime windowsdesktop`, then gets a different error message about `Microsoft.NETCore.App` because the one they have is not compatible and has to run the script again with `-Runtime dotnet`. (And hopefully, most people will be going through the installer instead.)

Reply:
That's a good point - makes sense to leave it out.